CHANGELOG.md

```diff
@@ -3,7 +3,47 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v2.0.0dev - [unreleased]
+## [v2.0.0](https://github.com/nf-core/pairgenomealign/releases/tag/2.0.0) "Naga imo" - [January 27th, 2025]
+
+### Backwards-incompatible changes
```

Comment:
These headers don't follow semi-conventional nf-core ones:

- `Added`
- `Fixed`
- `Dependencies`
- `Deprecated`

Some pipelines include `Changed`

You can see the customary headers in the template: https://github.com/nf-core/tools/blob/main/nf_core/pipeline-template/CHANGELOG.md

Reply:
Dependencies are often a table as you've done, but it's also customary to put the PR number (with a link) at the beginning of each entry, and tag the person who reported and then fixed/added the code

For example ampliseq: https://github.com/nf-core/ampliseq/blob/master/CHANGELOG.md

(however these are customs, not rules!)

Reply:
Well, this release is a single PR… but I will try to be more granular later and then I can mention that in the changelog.  Also I'm developing the pipeline alone for the moment so I will be shy and not credit myself. But In the next push I will udpate the headers so that they are closer to the example you gave.